src/Nethermind/Nethermind.Merge.Plugin/BlobTransactionsDbCleaner.cs

```diff
@@ -55,6 +58,10 @@ private void CleanProcessedTransactionsDb(long newlyFinalizedBlockNumber)
 
             if (_logger.IsDebug) _logger.Debug($"Cleaned processed blob txs from block {_lastFinalizedBlock} to block {newlyFinalizedBlockNumber}");
 
+            _db.Compact();
```

Comment:
Shall we only run compaction only after enough transactions were deleted from the db? Put some threshold?